design/common.blocks/button/_theme/button_theme_islands.styl

```diff
@@ -131,6 +131,11 @@ ctx = '.button_theme_islands'
                 background: #fee481;
             }
         }
+
+        &.button_disabled
+        {
+            background: 0;
```

Comment:
@tadatuta here we have to remove background at all. Also, `background-position-x: 0;` – it's default value.